test_suite.sh

```diff
@@ -201,13 +173,14 @@ else
 fi
 echo "Detected available memory: ${cgroup_mem_mib} MiB"
 
-echo "Replacing detected system information in template ReFrame config file..."
+# echo "Replacing detected system information in template ReFrame config file..."
```

Comment:
This line can probably be removed as well?

Reply:
Done